src/mesh/parallel/fci.cxx

```diff
@@ -132,14 +132,13 @@ FCIMap::FCIMap(Mesh& mesh, const Field2D& dy, Options& options, int offset_,
 
       xt_prime_corner[i] = -1.0;
       zt_prime_corner[i] = -1.0;
-      continue;
-    }
-
-    xt_prime_corner[i] =
-        0.25 * (xt_prime[i] + xt_prime[i_xplus] + xt_prime[i_zplus] + xt_prime[i_xzplus]);
+    } else {
+      xt_prime_corner[i] =
+          0.25 * (xt_prime[i] + xt_prime[i_xplus] + xt_prime[i_zplus] + xt_prime[i_xzplus]);
```

Comment:
I'm not sure I see any difference, this change is functionally identical? Am I missing something?

Reply:
Sorry, yes, that is just a cleanup - I thought the continue was confusing, if the else branch is so short.

Probably should have gone into another commit ...